recipe/windows-bat.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/graphviz/backend.py b/graphviz/backend.py
+index 955551a..be35745 100644
+--- a/graphviz/backend.py
++++ b/graphviz/backend.py
+@@ -69,6 +69,8 @@ def command(engine, format, filepath=None):
+     if format not in FORMATS:
+         raise ValueError('unknown format: %r' % format)
+ 
++    if PLATFORM == 'windows':
++        engine = '%s.bat' % engine
```

Comment:
It's not enough to change the name of the binary you want to call here (in fact, it's not even necessary). What you need to do (as I did in #2) is to tell `subprocess` that you want to run commands under a shell so that batch scripts are recognized as executables.